src/plugins/intel_gpu/src/kernel_selector/cl_kernels/fully_connected_gpu_bf_tiled.cl

```diff
@@ -327,7 +347,11 @@ inline void FUNC(fc_bf_tiled_kernel_default)(
 
             __local SLM_FILTER_VEC* slm_wei_vec = (__local SLM_FILTER_VEC*)wei_local_mem;
 
+            #if FILTER_LAYOUT_OS_IS_YX_OSV64_ISV2
+            uint weights_idx = weights_offset + local_id * SIMD * FILTER_LOAD_ITERS * FILTER_LOAD_BLOCK_SIZE * 2;
```

Comment:
Please fix the ACTUAL_LOAD_BLOCK_SIZE definition as it is done for os_iyx_osv16

Reply:
discussed it offline